website/content/docs/v0.9/Reference/cli.md

```diff
@@ -103,7 +103,7 @@ talosctl cluster create [flags]
       --ipv4                                    enable IPv4 network in the cluster (default true)
       --ipv6                                    enable IPv6 network in the cluster (QEMU provisioner only)
       --iso-path string                         the ISO path to use for the initial boot (VM only)
-      --kubernetes-version string               desired kubernetes version to run (default "1.20.5")
+      --kubernetes-version string               desired kubernetes version to run (default "1.21.0-beta.1")
```

Comment:
hell... we might need to hold on with this PR before we fork 0.10 docs. probably early next week, this change should go to 0.10 docs, but we don't have them yet until we actually do release 0.9.0